projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -409,13 +429,34 @@ release-image-build-on-metal-%: validate-supported-image-% $(GIT_PATCH_TARGET) $
 
 # These targets all end with <image format>-<os>-<os-version><optional -efi>
 # ex: local-build-ami-ubuntu-2004, s3-artifacts-ova-ubuntu-2004, upload-artifacts-ova-ubuntu-2004, upload-artifacts-ova-ubuntu-2004-efi
+# Special case for clone targets: local-clone-build-ova-redhat-9-efi
+local-clone-build-%: IMAGE_FORMAT=$(word 1,$(subst -, ,$*))
```

Comment:
Why can't we include this pattern in the below var assignment for the non-clone targets? The values are the same right?

Reply:
I think this is the easiest way even if we are doing some copy pasting work. I have tried using the same pattern and include `clone` in one of the positional args and it makes the logic very complicated. I have also tried to use pattern `local-build-%-clone%`, but then `clone` gets interpreted into os, provider or firmware

Reply:
Makes sense.